packages/grafana-ui/package.json

```diff
@@ -164,6 +163,7 @@
     "@types/tinycolor2": "1.4.3",
     "@types/uuid": "9.0.2",
     "common-tags": "1.8.2",
+    "core-js": "3.33.0",
```

Comment:
do we still need this?

Reply:
nope, good catch. also clean up another place.

Reply:
ah, never mind. we need them in devdeps for the tests which do `import 'core-js/stable/structured-clone';`